### PR TITLE
Feature/mobile tips tabs

### PR DESCRIPTION
**Pull request by cryptoB0T**
adding tabs for tips and coins on mobile #720708842260748

drawing

### CODE REVIEW

File: app/javascript/App/components/NewsfeedPage/SectionHeaderTight.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from 'react'
+
+const SectionHeaderTight = ({ children }) => {
+  console.log(children[0])
```
Comment (hanchang) [inline app/javascript/App/components/NewsfeedPage/SectionHeaderTight.js]: Let's nuke the console.log and tighten up the formatting!

File: app/javascript/App/components/NewsfeedPage/NewsListHeader.js [hunk 2]
```diff
@@ -1,30 +1,64 @@
...
+    padding: '18px',
+    borderRadius: 0
+  }
+
   return (
     <Fragment>
-      <SectionHeader>
-        <div className="flex-auto flex items-center">
+      {window.isMobile && (
+          <SectionHeaderTight>
```
Comment (hanchang) [inline app/javascript/App/components/NewsfeedPage/NewsListHeader.js]: Seems like the indentation is a bit weird

Comment (cryptoB0T) [reply app/javascript/App/components/NewsfeedPage/NewsListHeader.js]: thanks for pointing this out!

**Comment by cryptoB0T** [comment]
updated comp 
drawing

**Comment by cryptoB0T** [comment]
@hanchang  thanks for the review!

**Comment by cryptoB0T** [comment]
updated comp 
drawing

**Pull request merged**